dist/assets/crx-manifest.js-62f0816b.js.map

```diff
@@ -0,0 +1 @@
+{"version":3,"file":"crx-manifest.js-62f0816b.js","sources":["../../../../../@crx/manifest"],"sourcesContent":["export default \"{\\\"manifest_version\\\":3,\\\"name\\\":\\\"OpenAPI DevTools\\\",\\\"version\\\":\\\"1.5.2\\\",\\\"devtools_page\\\":\\\"index.html\\\",\\\"permissions\\\":[\\\"storage\\\"],\\\"icons\\\":{\\\"16\\\":\\\"icon16.png\\\",\\\"32\\\":\\\"icon32.png\\\",\\\"48\\\":\\\"icon48.png\\\",\\\"128\\\":\\\"icon128.png\\\"}}\""],"names":[],"mappings":"AAAA,iBAAe;;;;"}
```

Comment:
I don't think this is the way to add the built code to the repo

The original uses the [Releases](https://github.com/AndrewWalsh/openapi-devtools/releases) on Github - I think this is the way to go since this isn't code that should be in Github 😄

Reply:
Yep - realised this after doing it